resource.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * MyBB 1.9
+ * Copyright 2022 MyBB Group, All Rights Reserved
+ *
+ * Website: http://www.mybb.com
+ * License: http://www.mybb.com/about/license
+ *
+ */
+
+define('IN_MYBB', 1);
+
+define('NO_ONLINE', 1);
+
+define('THIS_SCRIPT', 'resource.php');
+
+// We'd prefer not to load this file, and instead load inc/init.php alone, because it is more
+// lightweight, but we need it for its resolution of the current theme for the current user, which
+// it sets $theme to, and which is used within resolve_themelet_resource().
+require_once './global.php';
```

Comment:
> This is only expected in "Development Mode".

Oh, ok, so only disabling otherwise seems to be missing I think.

Reply:
I meant to disable the `resource.php` file when the special mode is off, at least in the file itself I couldn't notice a code that blocks it. It might be on the `global.php` file or `init.php` so if so ignore my comment.